pkgs/development/libraries/libtickit/default.nix

```
@@ -18,25 +21,33 @@ stdenv.mkDerivation rec {
     hash = "sha256-QCrym8g5J1qwsFpU/PB8zZIWdM3YzOySknISSbQE4Sc=";
   };
 
-  makeFlags = [
-    "PREFIX=$(out)"
-    "LIBTOOL=${lib.getExe libtool}"
+  patches = [
+    # This test does not work on macos
```

I was hoping for a description of why it fails and thus why it is safe to ignore, sorry if i was not being clear. I skimmed the git logs and the test itself. Does this seem correct?

```suggestion
    # disabled on darwin, since test assumes TERM=linux
```

---

Indeed, It does.

---

I've forced push the description of the patch.